jekyll/_cci2/test-splitting-tutorial.adoc

```diff
@@ -113,19 +156,62 @@ steps:
     name: Run tests
     command: yarn test
 ----
+
+. After you've made these changes, go ahead and click the btn:[Commit and Run]. This will commit the changes in a new feature branch called `circleci-project-setup` and trigger a new pipeline. 
++
+image::test-splitting-first-pipeline.png[Successful pipeline run]
++
+Feel free to take a look at the steps run in the pipeline by expanding the green Success status and opening the `build-and-test` job.
++
+image::test-splitting-first-setup-steps.png[Steps run successfully within the job]
+--
+
+[.tab.add_project.GitLab]
+--
+. Import the project code into GitLab, using the repo URL: `https://github.com/CircleCI-Public/circleci-react-test-splitting-tutorial`
```

Comment:
Maybe we can add a banner along the lines of `If you are using GitLab self managed, jump to the full config and add this to your GitLab project, commit, push to see it build on CircleCI` ?